open_buildings/cli.py

```diff
@@ -74,7 +75,7 @@ def get_buildings(geojson_input, dst, source, country_iso, silent, overwrite, ve
     else:
         geojson_data = json.load(click.get_text_stream('stdin'))
```

Comment:
you could try something like this (untested)
```
if geojson_input:
    geojson_data = json.load(geojson_input)
elif geocode:
    geojson_data = mapping(geocode_to_wkt(geocode))
else:
    geojson_data = json.load(click.get_text_stream('stdin'))
```
here, we check whether someone has provided a geojson input. if so, we read it in. if they have instead provided the geocode option, we pass the string to your function which returns a shapely object. we obtain a geojson-like mapping using shapely.geometry.mapping. if neither of these two cases are true, we assume that geojson has been piped from stdin. 
this gives priority to geojson input, which means that if someones passes a geojson and uses the geocode option, the geocode option gets ignored. we could also have the function behave differently